pennylane/operation.py

```diff
@@ -102,18 +102,19 @@
 from enum import IntEnum
 from typing import List
 
+import autoray
 import numpy as np
 from numpy.linalg import multi_dot
 from scipy.sparse import coo_matrix, eye, kron
 
 import pennylane as qml
+from pennylane.interfaces import SUPPORTED_INTERFACES
```

Comment:
The circular import was originated here. Should we move `SUPPORTED_INTERFACES` to the `pennylane/__init__.py` file instead?